Package.swift

```diff
@@ -0,0 +1,62 @@
+// swift-tools-version:5.5
+
+import PackageDescription
+
+let package = Package(
+    name: "WordPressUI",
+    platforms: [.iOS(.v11)],
+    products: [
+        .library(
+            name: "WordPressUI",
+            targets: [
+                "WordPressUIGravatar",
+                "WordPressUIGravatarObjC",
+                "WordPressUIObjCCategories",
+                "WordPressUI",
+            ]
+        )
+    ],
+    dependencies: [],
+    targets: [
+        .target(
+            name: "WordPressUIGravatar",
+            path: "WordPressUI/Extensions/Gravatar",
+            sources: ["Gravatar.swift"]
+        ),
+        .target(
+            name: "WordPressUIGravatarObjC",
+            path: "WordPressUI/Extensions/Gravatar",
```

Comment:
@mokagio The warning about `NSString+Gravatar.m` not included in any target is a bit strange, as it should be included in this target. But I have a follow up PR to reorganize the folder structure, that warning will go away since the targets will also change.

Reply:
👍